orbit/.goreleaser.yml

```diff
@@ -22,9 +21,30 @@ builds:
       - -X github.com/fleetdm/fleet/v4/orbit/pkg/build.Version={{.Version}}
       - -X github.com/fleetdm/fleet/v4/orbit/pkg/build.Commit={{.Commit}}
       - -X github.com/fleetdm/fleet/v4/orbit/pkg/build.Date={{.Date}}
+
+  - id: orbit-macos
+    dir: ./orbit/cmd/orbit/
+    binary: orbit
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
```

Comment:
Is this a feature of goreleaser? (as in: it builds one binary provided two flags here)

Reply:
Ah indeed: https://goreleaser.com/customization/universalbinaries/